src/include/cell_inline.h

```diff
@@ -38,6 +38,14 @@ __cell_pack_value_validity(WT_SESSION_IMPL *session, uint8_t **pp, WT_TIME_WINDO
 {
     uint8_t flags, *flagsp;
 
+    WT_ASSERT(session,
```

Comment:
This assert is the same as the one below, consider creating a function so we can change in one place.